includes/html/functions.inc.php

```diff
@@ -936,6 +936,17 @@ function alert_details($details)
             $fallback = false;
         }
 
+        if ($tmp_alerts['isisISAdjIPAddrAddress']) {
```

Comment:
What is the reason for this special case?

Reply:
To format the alert to show only the adjacents and not general polling time or device's uptime when expanded. See the screenshot.
![isis_special_case](https://user-images.githubusercontent.com/7031571/106005504-f8401700-60bc-11eb-819c-420a7cccf829.PNG)

Reply:
Hmm, this feels like the wrong place for that, but I don't have any help beyond that, sorry.

Reply:
I suggest  moving this code to a separate PR, it is going to be a blocker for this PR.

Reply:
OK, I will remove this and make a separate PR for this later.